config/src/config/consensus_config.rs

```diff
@@ -35,8 +35,8 @@ pub struct ConsensusConfig {
 impl Default for ConsensusConfig {
     fn default() -> ConsensusConfig {
         ConsensusConfig {
-            max_block_txns: 3500,
-            max_block_bytes: 5 * 1024 * 1024, // 5MB
+            max_block_txns: 2500,
+            max_block_bytes: 600 * 1024, // 600 KB
```

Comment:
Can you clarify where did you get this number from? If its from running P2P txns in Forge, we might want a higher threshold as other types of txns size might be larger.

Reply:
As discussed offline, let's add a comment with back-of-the-envelope calculation for coming up with this number.